sass/components/_emoticons.scss

```diff
@@ -628,7 +628,10 @@
     }
 
     .emoji-picker__preview-image {
+        left: 2px;
         max-height: 36px;
-        max-width: 42px;
+        max-width: 36px;
+        position: relative;
+        top: -8px;
     }
```

Comment:
What are these changes for?

Reply:
See https://github.com/mattermost/mattermost-webapp/pull/8309#discussion_r658189777. You beat me to writing that up 😛